lib/vsc/install/shared_setup.py

```diff
@@ -756,7 +756,6 @@ def action_target(target, setupfn=setup, extra_sdist=[], urltemplate=None):
         'version': VERSION,
         'author': [sdw, ag, jt],
         'maintainer': [sdw, ag, jt],
-        'zip_safe': True,
```

Comment:
as mentioned, I would keep this included, set to `False`, and include a comment why